src/app/shared/add-entity-popup/add-entity-popup.component.css

```
@@ -0,0 +1,147 @@
+    display: flex;
+    margin-top: 0.5rem;
+    justify-content: center;
+}
+
+.srch .mat-mdc-text-field-wrapper {
+    height: 2.5rem !important;
+    align-items: center;
+    border-radius: 0.5rem !important;
+}
+
+::ng-deep {
```

Biome (2.1.2)

[error] Unexpected unknown pseudo-elements: ng-deep

See MDN web docs for more details.
Use a known pseudo-elements instead, such as:


(lint/correctness/noUnknownPseudoElement)